Team9ProjectTests/src/com/indragie/comput301as1/test/ReceiptTests.java

```diff
@@ -0,0 +1,17 @@
+package com.indragie.comput301as1.test;
+
+import com.indragie.cmput301as1.ExpenseItemAddActivity;
+
+import android.test.ActivityInstrumentationTestCase2;
+
+public class ReceiptTests extends ActivityInstrumentationTestCase2<ExpenseItemAddActivity> {
+
+	public ReceiptTests(String name) {
```

Comment:
I don't think you need to test the part that writes the files. If you create a method that just does the scaling (and returns the `Bitmap` object) it would be simple to write a test that verifies that the bitmap dimensions don't exceed the limits.